ckanext/geonode/harvesters/client.py

```diff
@@ -35,13 +34,9 @@ def _get_resources(self, resType):
         url = '%s/api/%s/' % (self.baseurl, resType)
 
         log.info('Retrieving %s at GeoNode URL %s', resType, url)
-        request = urllib2.Request(url)
-        opener = urllib2.build_opener(urllib2.HTTPCookieProcessor(), urllib2.HTTPRedirectHandler())
-
-        response = opener.open(request)
-        content = response.read()
+        response = urlopen(url)
 
-        json_content = json.loads(content)
+        json_content = json.loads(response)
```

Comment:
This will throw a `the JSON object must be str, bytes or bytearray, not HTTPResponse`